src/ui/ObjectControls.js

```diff
@@ -68,12 +68,14 @@ ObjectHandler.prototype.setObjects = function (objects) {
 
 ObjectHandler.prototype.rotate = function (mousePrevPos, mouseCurPos, aboutAxis) {
   const rot = this.mouse2rotation(mousePrevPos, mouseCurPos, aboutAxis);
+  const quat = new THREE.Quaternion().setFromAxisAngle(rot.axis, rot.angle);
```

Comment:
I see you've left a quaternion creation that has already existed. But can we refactor the function using closure? What do you think?

Reply:
I think it's great idea
Done)